Python/tests/conftest.py

```diff
@@ -1,12 +1,14 @@
 import os
-from sqlmlutils import ConnectionInfo
+from sqlmlutils import ConnectionInfo, Scope
 
 driver = os.environ['DRIVER'] if 'DRIVER' in os.environ else "SQL Server"
 server = os.environ['SERVER'] if 'SERVER' in os.environ else "localhost"
 database = os.environ['DATABASE'] if 'DATABASE' in os.environ else "AirlineTestDB"
 uid = os.environ['USER'] if 'USER' in os.environ else ""
 pwd = os.environ['PASSWORD'] if 'PASSWORD' in os.environ else ""
 
+scope = Scope.public_scope() if uid == "" else Scope.private_scope()
```

Comment:
nit: are we on purpose handling empty string "" as the only replacement for public_scope? for instance, what if uid is None? Or any boolean or other type for this matter.

Reply:
connectionInfo is defined like this:

 def __init__(self, driver: str = "SQL Server", server: str = "localhost", port: str = "", database: str = "master", uid: str = "", pwd: str = ""):

We have type annotations for each of the variables so I'm not sure we need to handle other types. Do you think we should add extra checking in the init function?